apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -1,18 +1,27 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import DCDO from '@cdo/apps/dcdo';
+
 import ResponseMenuDropdown from './ResponseMenuDropdown';
 
 import styles from './summary.module.scss';
 
-const FreeResponseResponses = ({responses}) => {
+const FreeResponseResponses = ({responses, showStudentNames}) => {
   return (
     <div className={styles.studentResponsesColumns}>
       {responses.map(response => (
-        <div key={response.user_id} className={styles.studentAnswer}>
-          <div className={styles.studentAnswerInterior}>
-            <p>{response.text}</p>
-            <ResponseMenuDropdown response={response} />
+        <div key={response.user_id} className={styles.studentResponseBlock}>
+          <div className={styles.studentAnswer}>
+            <div className={styles.studentAnswerInterior}>
+              <p>{response.text}</p>
+              <ResponseMenuDropdown response={response} />
+            </div>
+            {DCDO.get('cfu-show-names-enabled', false) && (
+              <div className={styles.studentName}>
+                {showStudentNames && <p>{response.student_name}</p>}
```

Comment:
Let's move `showStudentNames &&` outside the studentName div so we aren't creating an empty div when we don't need to. Unless we need it for margins?

Reply:
Also I don't think student_name should be in a <p> tag since it's only one name (<p> for paragraph). Could we remove?

Reply:
This was my solution for getting the names to show and un-show without re-locating the text responses (shifting them up and down as the names got entered). 

For the `<p>` tag, I will still need to wrap this in something (a `<div>`?).  Is there a reason in your mind that a `<div>` would be better than `<p>`?